sensor_msgs/Image img
sensor_msgs/Image depthImg
bounding_box box
